api/v1beta2/foundationdbcluster_types.go

```diff
@@ -1677,24 +1677,27 @@ type RoutingConfig struct {
 	DNSDomain *string `json:"dnsDomain,omitempty"`
 }
 
-// RequiredAddressSet provides settings for which addresses we need to listen
+// Requiredaddressset provides settings for which addresses we need to listen
 // on.
 type RequiredAddressSet struct {
 	// TLS defines whether we need to listen on a TLS address.
 	TLS bool `json:"tls,omitempty"`
 
 	// NonTLS defines whether we need to listen on a non-TLS address.
-	NonTLS bool `json:"nonTLS,omitempty"`
+	NonTLS bool `jsogn:"nonTLS,omitempty"`
 }
 
+// ProcessGroup -> ContainerName
+type CrashLoopList map[string](map[string]None)
```

Comment:
We shouldn't use maps here and the format should look like specified in the issue: https://github.com/FoundationDB/fdb-kubernetes-operator/issues/1438:

```yaml
buggify:
  crashLoopContainer:
  - container: foundationdb
     targets:
     - storage-1
     - storage-2
  - container: foundationdb-kubernetes-sidecar
     targets:
     - storage-3
```

Some background why maps shouldn't be used in Kubernetes structs: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#lists-of-named-subobjects-preferred-over-maps